### ISSUE TITLE
Breakage on Minecraft 1.19

### ISSUE THREAD

**Issue by haykam821** [VelocityPowered/CrossStitch#7]
The mixin likely needs to be moved to the `CommandTreeS2CPacket.ArgumentNode.write` method. Additional logging would be beneficial here for mods that do not register their argument types to the class map because of silent breakage introduced by the argument type refactor (mods should register using [`ArgumentTypeRegistry`](https://github.com/FabricMC/fabric/blob/e1dbfca1b3abcb3614f52ac5cec24941da47c69f/fabric-command-api-v2/src/main/java/net/fabricmc/fabric/api/command/v2/ArgumentTypeRegistry.java) instead).

Note that argument types now sync using raw IDs as well, and Velocity seems to represent `crossstitch:mod_argument` as the raw ID `-255`.

**Comment by LaurenceBarnes** [comment]
I got this error spamming every few seconds, could that be related? https://pastebin.com/W9npq6dV

**Comment by haykam821** [comment]
Yes, that mixin error is due to the argument type refactor. The pull request #8 which I opened fixes that error by moving the serialization to the new inner class.
